app/pages/editorg/template.njk

```diff
@@ -0,0 +1,54 @@
+{% extends 'includes/layout.njk' %}
+{% from 'components/back-link/macro.njk' import backLink %}
+{% from 'components/button/macro.njk' import button %}
+{% from 'components/checkboxes/macro.njk' import checkboxes %}
+{% from '../../common/components/table.njk' import table %}
+
+{% block body %}
+  {% if agreementSigned %}
+    {% set checkedValue %}checked{% endset %}
+  {% endif %}
+
+  <div class="nhsuk-width-container" data-test-id="edit-organisation-page">
+    <div data-test-id="go-back-link">
+      {{ backLink({
+        "href": backLinkHref,
+        "text": "Back"
+      }) }}
+    </div>
+  
+    <h1 data-test-id="edit-org-page-title" class="nhsuk-u-margin-top-7 nhsuk-u-margin-bottom-3">Edit {{ organisationName }}</h1>
+    <h2 data-test-id="edit-org-page-description" class="nhsuk-u-font-size-22 nhsuk-u-margin-bottom-9">{{ description }}</h2>
+
+    <h3 data-test-id="organisation-ods-code-heading" class="nhsuk-heading-s nhsuk-u-margin-bottom-3">{{ odsCodeHeading }}</h3>
+    <div data-test-id="organisation-ods-code" class="nhsuk-u-font-size-19 nhsuk-u-margin-bottom-5">{{ odsCode }}</div>
+    <h3 data-test-id="organisation-name-heading" class="nhsuk-heading-s nhsuk-u-margin-bottom-3">{{ organisationNameHeading }}</h3>
+    <div data-test-id="organisation-name" class="nhsuk-u-font-size-19 nhsuk-u-margin-bottom-5">{{ organisationName }}</div>
+    <h3 data-test-id="organisation-address-heading" class="nhsuk-heading-s nhsuk-u-margin-bottom-3">{{ addressHeading }}</h3>
+    <div data-test-id="organisation-address" class="nhsuk-u-margin-bottom-5">
+      {% for line in address %}
+        <div data-test-id="organisation-address-{{ loop.index }}" class="nhsuk-u-font-size-19">{{ line }}</div>
+      {% endfor %}
+    </div>
+    <h3 data-test-id="organisation-primary-role-id-heading" class="nhsuk-heading-s nhsuk-u-margin-bottom-2">{{ primaryRoleIdHeading }}</h3>
+    <div data-test-id="organisation-primary-role-id" class="nhsuk-u-font-size-19 nhsuk-u-margin-bottom-5">{{ primaryRoleId }}</div>
+
+    <form method="post" action="{{ saveCatalogueAgreementPostAction }}">
+      <input type="hidden" name="_csrf" value="{{ csrfToken }}" />
+    
+      <div class="nhsuk-checkboxes__item nhsuk-u-margin-bottom-9">
+        <input data-test-id="catalogue-agreement-signed-checkbox" class="nhsuk-checkboxes__input" id="catalogue-agreement-checkbox" name="catalogueagreementsigned" type="checkbox" value="{{agreementSigned}}" {{ checkedValue }}>
```

Comment:
any reason why we are not using the NHS checkbox component instead?

Reply:
it doesn't seem to allow it to be pre checked?

Reply:
really? I think it MP we use that component and check it. Let me have a look what we do there

Reply:
You need to pass in `checked: true` for the items property in that component.

Reply:
O I C 👀